terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
```

Comment:
nit: add version strings to these providers. I'm pretty sure it tells you exactly what strings to add when you run `terraform init`

Reply:
Version strings added

Reply:
Version added